src/components/Footer/ListLinks.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Link from './Link';
+
+const ListLinks = ({classNameContainer, classNameChild, children, links}) => (
+  <div className={classNameContainer}>
+    {children}
```

Comment:
Do we really want to let people pass children besides array of links?

Reply:
fixed